libs/server/Resp/RespServerSession.cs

```diff
@@ -901,7 +943,7 @@ private void Send(byte* d)
             if ((int)(dcurr - d) > 0)
             {
                 // Debug.WriteLine("SEND: [" + Encoding.UTF8.GetString(new Span<byte>(d, (int)(dcurr - d))).Replace("\n", "|").Replace("\r", "!") + "]");
-                if (storeWrapper.appendOnlyFile != null && storeWrapper.serverOptions.WaitForCommit)
+                if (runningWithAOFWaitForCommitMode && waitForAofBlocking)
```

Comment:
This can test just waitForAofBlocking, as it is not set true if runningWithAOFWaitForCommitMode is false

Reply:
nice! completely missed the redundancy. Thanks